source/MainPanel.cpp

```diff
@@ -500,6 +459,128 @@ bool MainPanel::ShowHailPanel()
 
 
 
+bool MainPanel::ShowHelp(bool force)
+{
+	const Ship *flagship = player.Flagship();
+	if(!flagship)
+		return false;
+
+	vector<string> forced;
+	// Check if any help messages should be shown.
+	if(Preferences::Has("Control ship with mouse"))
+	{
+		if(force)
+			forced.push_back("control ship with mouse");
+		else
+			return DoHelp("control ship with mouse");
```

Comment:
I think this method of returning the result of DoHelp means that most of these help dialogs will never show. The previous method would evaluate DoHelp and if it returned false it would move onto the next help dialog. With this, if you have the above preference, this is immediately returning the result of DoHelp for this tutorial without bothering to check anything else.